x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/calendars/ScheduledEventTests.java

```diff
@@ -29,7 +29,8 @@
 public class ScheduledEventTests extends AbstractSerializingTestCase<ScheduledEvent> {
 
     public static ScheduledEvent createScheduledEvent(String calendarId) {
-        ZonedDateTime start = ZonedDateTime.ofInstant(Instant.ofEpochMilli(new DateTime(randomDateTimeZone()).getMillis()), ZoneOffset.UTC);
+        long millis = Clock.system(randomZone()).millis();// TODO we are converting to UTC anyway??
+        ZonedDateTime start = Instant.ofEpochMilli(millis).atZone(ZoneOffset.UTC);
```

Comment:
not sure we need to do soo many convertions here. maybe using just 'ZonedDateTime.now(UTC)` ?

Reply:
ack. can you check why we need random timezone here at all?